### PR TITLE
[ONNX] Add search of tensors in nodes attributes; Add pattern

### PR DESCRIPTION
**Pull request by kshpv**
### Changes

Add a function to properly search tensors in ONNX model; Add fused pattern for correct quantization scheme for a model from  https://github.com/openvinotoolkit/nncf/issues/1936

(Left) **Correct** scheme with the suggested changes |            (Right) **Incorrect** scheme on develop branch:
image image

### Reason for changes

To enable a model from issue https://github.com/openvinotoolkit/nncf/issues/1936

### Related tickets

No

### Tests

No

### CODE REVIEW

File: nncf/onnx/hardware/fused_patterns.py [hunk 1]
```diff
@@ -82,6 +82,25 @@ def create_swish_with_hard_sigmoid() -> GraphPattern:
     return pattern
 
 
+@ONNX_HW_FUSED_PATTERNS.register(HWFusedPatternNames.HSWISH_ACTIVATION)
```
Comment (andrey-churkin) [inline nncf/onnx/hardware/fused_patterns.py:85]: Could you please why you did not include the `Div` operation here? As I remember, the h-shish is
```math
hswish(x) = x \frac{ReLU6(x+3)}{6}
```

File: nncf/onnx/hardware/fused_patterns.py [hunk 2]
```diff
@@ -256,6 +275,16 @@ def create_linear_batch_norm_activations() -> GraphPattern:
     return linear_batch_norm
 
 
+@ONNX_HW_FUSED_PATTERNS.register(HWFusedPatternNames.LINEAR_BATCH_NORM_ACTIVATIONS_ARITHMETIC)
```
Comment (andrey-churkin) [inline nncf/onnx/hardware/fused_patterns.py:278]: It is not clear for me why this fused pattern was added. Could you please briefly explain the reason?

File: nncf/onnx/hardware/fused_patterns.py (hunk 1 above)
Comment (kshpv) [reply nncf/onnx/hardware/fused_patterns.py:85]: You are right

File: nncf/onnx/hardware/fused_patterns.py (hunk 2 above)
Comment (kshpv) [reply nncf/onnx/hardware/fused_patterns.py:278]: Sure, I incorrectly didn't add a Div layer in HSwish pattern. If HSwish pattern has Div op, there is no need in `LINEAR_BATCH_NORM_ACTIVATIONS_ARITHMETIC` pattern

File: tests/onnx/quantization/test_bias_correction.py [hunk 3]
```diff
@@ -20,6 +20,8 @@
 from nncf.quantization.algorithms.bias_correction.onnx_backend import ONNXBiasCorrectionAlgoBackend
 from tests.post_training.test_templates.test_bias_correction import TemplateTestBCAlgorithm
 
+#
+
```
Comment (l-bat) [inline tests/onnx/quantization/test_bias_correction.py]: ?

File: tests/onnx/test_pattern_manager.py [hunk 4]
```diff
@@ -27,7 +27,6 @@
     HWFusedPatternNames.SE_BLOCK: "Not relevant for ONNX.",
     HWFusedPatternNames.SOFTMAX_DIV: "Not relevant for ONNX.",
     HWFusedPatternNames.EQUAL_LOGICALNOT: "Not relevant for ONNX.",
-    HWFusedPatternNames.HSWISH_ACTIVATION: "Not relevant for ONNX.",
     HWFusedPatternNames.HSWISH_ACTIVATION_V2: "Not relevant for ONNX.",
```
Comment (l-bat) [inline tests/onnx/test_pattern_manager.py:30]: Are you sure that `HWFusedPatternNames.HSWISH_ACTIVATION_V2` or `HWFusedPatternNames.HSWISH_ACTIVATION_WITHOUT_DENOMINATOR` are still not relevant for ONNX?

File: nncf/onnx/hardware/fused_patterns.py (hunk 1 above)
Comment (kshpv) [reply nncf/onnx/hardware/fused_patterns.py:85]: Updated

Comment (kshpv) [reply nncf/onnx/hardware/fused_patterns.py:85]: I checked out that the pattern applied correctly and the quantization scheme is correct on the particular model.

File: tests/onnx/quantization/test_bias_correction.py (hunk 3 above)
Comment (kshpv) [reply tests/onnx/quantization/test_bias_correction.py]: This was removed

File: tests/onnx/test_pattern_manager.py (hunk 4 above)
Comment (kshpv) [reply tests/onnx/test_pattern_manager.py:30]: I don't have evidence that we should add them for ONNX now

Comment (kshpv) [reply tests/onnx/test_pattern_manager.py]: Added `HSWISH_ACTIVATION_WITHOUT_DENOMINATOR`; add comment about `HSWISH_ACTIVATION_V2 `

**Review by l-bat — APPROVED** [review]

**Comment by kshpv** [comment]
run pytorch pre-commit tests
run pylint pre-commit tests

**Comment by kshpv** [comment]
run pylint pre-commit tests

**Pull request merged**